server/sonar-server/src/test/java/org/sonar/server/organization/ws/DeleteActionTest.java

```diff
@@ -348,6 +349,25 @@ public void delete_quality_profiles_of_specified_organization() {
       .containsOnly(profileInOtherOrg.getKee());
   }
 
+  @Test
+  public void delete_quality_gates() {
+    OrganizationDto org = db.organizations().insert();
+    OrganizationDto otherOrg = db.organizations().insert();
+    QGateWithOrgDto qualityGate = db.qualityGates().insertQualityGate(org);
+    QGateWithOrgDto qualityGateInOtherOrg = db.qualityGates().insertQualityGate(otherOrg);
+    logInAsAdministrator(org);
+
+    sendRequest(org);
```

Comment:
I think here we should check that built-in is not destroyed.

Reply:
You can add an UT, but the check already exist in the code (or it should if it's not the case)